premake5.lua

```diff
@@ -10,21 +10,28 @@ BUILDDIR = _OPTIONS["builddir"] or "build"
 
 workspace "wxsqlite3"
   configurations { "Debug", "Release", "Debug wxDLL", "Release wxDLL", "Debug DLL", "Release DLL" }
-  platforms { "Win32", "Win64" }
+  platforms { "Win32", "Win64", "MacOsx" }
```

Comment:
Premake has no predefined platforms. Here platform names could be for example something like `MacOS_x64` or `MacOS_ARM64`.